cli/src/utils/user-config.ts

```diff
@@ -0,0 +1,13 @@
+import { join } from 'path';
+import { homedir } from 'os';
+import Conf from 'conf';
+import { safeDump, safeLoad } from 'js-yaml';
+
+export default new Conf({
+  fileExtension: 'yml',
```

Comment:
Why do we care if it's yaml if we don't intend for it to be edited manually?
Is this part of the spec?

Reply:
yes
> When the user successfully completes the process, logic will create a credentials file at `~/.shiftjs/shiftjs.config.yml` which will store the authenticated credentials.